helm-chart/templates/analytics/ingress.yaml

```diff
@@ -1,48 +1,38 @@
 {{- $version := .Capabilities.KubeVersion.GitVersion -}}
 {{- if and .Values.ingress.enabled .Values.analytics.enabled -}}
 {{- $fullName := include "cvat.fullname" . -}}
-{{- if semverCompare ">=1.19-0" $version -}}
```

Comment:
You should add a [`kubeVersion`](https://helm.sh/docs/topics/charts/#the-chartyaml-file) field to `Chart.yml` to declare supported Kubernetes versions. This should also be reflected in the changelog.